CHANGELOG.md

```diff
@@ -82,6 +75,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Removed operator splitting in CH4 simulation that was biasing diagnostics
 - Fixed GCHP start and elapsed times in time_mod.F90 to use cap_restart value
 - Disabled SpeciesConcMND output for benchmark simulations
+- Exit `Init_Photolysis` before calling `Set_Aer` when doing dry-run simulations
+
+### Removed
+- `Warnings: 1` is now removed from `HEMCO_Config.rc.*` template files
+- Removed the `NcdfUtil/perl` folder
+- Removed `X-HRS` output from log file
+- IONO2 recycling (fullchem, custom mechanisms)
+- Deleted unused file set_prof_o3.F90
```

Comment:
None of the updates listed above are in this PR.

Reply:
I've moved the whole "Removed" section from above, so that the order in the changelog is "Added", "Changed", "Fixed", "Removed".  I'll add a note in the commit.

Reply:
Added a commit message stating such in 631a8962,